R/make-array.R

```diff
@@ -111,11 +111,12 @@ makeMRFromText <- function (var,
     if (missing(name)) {
         halt("Must supply a name for the new variable")
     }
-    if (is.Categorical(var) || is.Text(var)) {
+    if (is.Text(var)) {
         uniques <- names(table(var))
     } else {
-        halt(dQuote(substitute(var)),
-             " must be a Categorical or Text Crunch Variable.")
+        halt(dQuote(deparse(substitute(var))),
```

Comment:
I'm curious what the `deparse` is doing here? I was under the impression that `substitute` returned a character-enough object for `halt`/`stop` to deal with. Was there an issue without it?

Reply:
Aaaah, makes sense. TIL!